### DISCUSSION TITLE
Development process

### DISCUSSION THREAD

**Discussion by Theselman** [hm21/pro_image_editor#100] [Q&A] [answered]
### Please use the discuss tab to ask a question.

Thanks for improving the package and continuing. I wonder if you will continue to develop it, for example, will you continue to update it after 6 months?

**Comment by hm21** [comment, marked as answer]
It is hard to say what will happen in the future because it depends on many factors. Just as an example, below is a short list with some factors.

- There is a new framework better than Flutter, which makes more sense to develop there.
- Someone releases an image editor which is much better than this editor, which makes it useless.
- AI is so smart that it generates a better image generator depending on what the user requires.
- My private situation change that I have no time to develop or, in the worst case, I pass away.
- A company makes me an offer to develop this editor for them alone in a much better version, will stop that I continue the progress here.

And of course, there are more factors that can end that I stop working on this project. 

However, in the case that I will not work on this project anymore in the future, I will possibly give the editing rights to someone else or transfer it to another user who wants to continue. In the worst-case someone can also fork it and continue it, so don't worry, I'm sure this project will be up-to-date also in the upcoming months.

FYI: I would love it if you would post your question in the [Discussion](https://github.com/hm21/pro_image_editor/discussions) tab next time because it makes it easier for other users to search for a question if we all have it there.
